biblib/views/user_view.py

```diff
@@ -154,6 +154,9 @@ def get(self):
         HTTP GET request that returns all the libraries that belong to a given
         user
 
+        :param start: The index of the library list to start on (int).  default: 0
+        :param rows: The number of rows to return from the start point (int). default: None
```

Comment:
Can you clarify that rows=None will return all rows?

Reply:
Added a line to make what none does more explicit.